apps/web/src/Providers.tsx

```diff
@@ -20,7 +20,6 @@ const queryClient = new QueryClient({
     queries: {
       queryFn: defaultQueryFn as any,
       refetchOnWindowFocus: false,
```

Comment:
Refetch on window focus is already disabled on this line, it’s not clear what this change is achieving by removing the retry option. Could you elaborate?

Reply:
Please ignore. This change was pushed as part of the PW migration.